src/lib.rs

```diff
@@ -444,8 +449,8 @@ impl<'a> SelfEncryptor<'a> {
 mod test {
   use super::*;
 
-  fn random_string(length: u64) -> String {
-        (0..length).map(|_| (0x20u8 + (super::rand::random::<f32>() * 96.0) as u8) as char).collect()
+  fn random_bytes(length: u64) -> Vec<u8> {
```

Comment:
If this is now random bytes, is there any requirement for them to still be in the range 0x20..0x20+96?